### PR TITLE
Added Glassmorphism #79

### LINKED ISSUE #79: Differrent GLassmorphism Card UI Design with Vanila
I am a contributor from GSSoC - 22. I can create different kinds of glassmorphism card UI designs like glowing gradients and etc. and can add some effects like hover and tilt effects.so Please assign this issue to me

https://user-images.githubusercontent.com/97339865/156155012-4f9c5ee4-b50f-4398-9f27-893ca349870a.mp4

### PR DESCRIPTION
**Pull request by CoderSahil128**
<hr>

## Description 📜

I am a contributor from GSSoC - 22. I have created different kinds of glassmorphism card UI designs and have added some effects like hover and tilt effects.so So kindly Review and merge my PR.

Fixes #79 

<hr>

## Type of change 📝

Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this

- [x] Add a new project

<hr>

## Domain of Contribution 📊

Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this

- [ ] Project using only HTML & CSS
- [x] Project using HTML, CSS & JS
- [ ] Project using ReactJs

<hr>
 
## Checklist ✅

Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this

- [x] I have performed a self-review of my own code or work.
- [x] My changes generates no new warnings.
- [x] I'm GSSoC'22 contributor

<hr>
 
## Screenshots / Gif (Optional) 📸

Add the screenshot of the changes you have made

Demo1
Demo2
Demo3

### CODE REVIEW

File: Animated-Glassmorphism-Vanila-Js/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<h1>Simon Game</h1>
+
+<h2>Overview</h2>
+
+<p>This application is made for beginners.User can use this web app to play Simon Game.</p>
+
+<h3>Tech Stack</h3>
```
Comment (meinhoonharsh) [inline Animated-Glassmorphism-Vanila-Js/README.md:7]: Writing Heading using `#` in Markdown would be a Good practice

File: Animated-Glassmorphism-Vanila-Js/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+![HTML](https://img.shields.io/badge/html5%20-%23E34F26.svg?&style=for-the-badge&logo=html5&logoColor=white)
+![CSS](https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white)
+![JS](https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E)
+
+## Screenshots
+![Demo1](https://raw.githubusercontent.com/CoderSahil128/WebDev-ProjectKart/master/Animated-Glassmorphism-Vanila-Js/Demo1.jpeg)
+![Demo2](https://raw.githubusercontent.com/CoderSahil128/WebDev-ProjectKart/master/Animated-Glassmorphism-Vanila-Js/Demo2.jpeg)
+![Demo3](https://raw.githubusercontent.com/CoderSahil128/WebDev-ProjectKart/master/Animated-Glassmorphism-Vanila-Js/Demo3.jpeg)
```
Comment (meinhoonharsh) [inline Animated-Glassmorphism-Vanila-Js/README.md:16]: Giving Relative path in the same Repository will be a better option, If you are using Path from your Repository, then If someday you privates/delete your repository, then these Images will be no more accessible

**Review by meinhoonharsh — CHANGES_REQUESTED** [review]
All things are pretty good, just requested few changes in Readme

**Comment by khushi-purwar** [comment]
Kindly do the changes requested by @meinhoonharsh

**Comment by gurjeetsinghvirdee** [comment]
No response from the contributor's side so I'm closing this issue

**Pull request closed without merging**